cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -411,15 +411,19 @@ private class CampaignManagerImpl(
             val localStateSize = messagesOptimizedLocalState?.jsonObject?.size ?: 0
             val isV6LocalStatePresent = dataStorage.preference.all.containsKey(LOCAL_STATE)
             val isV6LocalStatePresent2 = dataStorage.preference.all.containsKey(DataStorage.LOCAL_STATE_OLD)
-            val res =
-                ((gdprUUID != null || ccpaUUID != null) && localStateSize == 0) || isV6LocalStatePresent || isV6LocalStatePresent2
+            val isRequireNewConsentData = dataStorage.localDataVersion < DataStorage.HARDCODED_LOCAL_DATA_VERSION
```

Comment:
I'd use the operator `!=` instead of `<`. I can't imagine the local version ever being higher than the hardcoded one, but it doesn't hurt...

Reply:
@andresilveirah I agree with you, it will be a good sanity check